fake-files/output-expected/vanderbilt_MxIF/vanderbilt_MxIF.images/vanderbilt_MxIF.tiff.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
Is there a reason the input `vanderbilt.ome.tif` has one "f" and outputs have two?

Reply:
I think the two are interchangeable so presumably libvips has some sort of preference that is being revealed here.